diracx-db/src/diracx/db/sql/pilot_agents/db.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+from datetime import datetime, timezone
+
+from sqlalchemy import insert
+
+from ..utils import BaseSQLDB
+from .schema import PilotAgents, PilotAgentsDBBase
+
+
+class PilotAgentsDB(BaseSQLDB):
+    metadata = PilotAgentsDBBase.metadata
+
+    async def addPilotReferences(
```

Comment:
I forgot this one, sorry: 

```suggestion
    async def add_pilot_references(
```